config/300-kafkasource.yaml

```diff
@@ -0,0 +1,399 @@
+                            type: object
+                            properties:
+                              name:
+                                type: string
+                              key:
+                                type: string
+                            required:
+                            - name
+                            - key
+                        required:
+                        - valueFromSecret
+                  sslAuth:
+                    description: SSL Authentication method to interact with Kafka.
+                    type: object
+                    properties:
+                      sslCA:
+                        description: The value to the configured CA.
+                        type: object
+                        properties:
+                          valueFromSecret:
+                            description: A reference to a Kubernetes Secret object containing the value.
+                            type: object
+                            properties:
+                              name:
+                                type: string
+                              key:
+                                type: string
+                            required:
+                            - name
+                            - key
+                        required:
+                        - valueFromSecret
+                      sslClientCert:
+                        description: The value of the SSL Client Cert.
+                        type: object
+                        properties:
+                          valueFromSecret:
+                            description: A reference to a Kubernetes Secret object containing the value.
+                            type: object
+                            properties:
+                              name:
+                                type: string
+                              key:
+                                type: string
+                            required:
+                            - name
+                            - key
+                        required:
+                        - valueFromSecret
+                      sslClientKey:
+                        description: The value of the SSL Client Key.
+                        type: object
+                        properties:
+                          valueFromSecret:
+                            description: A reference to a Kubernetes Secret object containing the value.
+                            type: object
+                            properties:
+                              name:
+                                type: string
+                              key:
+                                type: string
+                            required:
+                            - name
+                            - key
+                        required:
+                        - valueFromSecret
+                      insecureSkipVerify:
```

Comment:
we are calling this `skipVerify` at HTTP Poller and Elasticsearch manifests. If you feel this is a better description, stick to it.